redis/db/redisbp/hooks.go

```diff
@@ -79,20 +82,54 @@ func (h SpanHook) startChildSpan(ctx context.Context, cmdName string) context.Co
 		name,
 		tracing.SpanTypeOption{Type: tracing.SpanTypeClient},
 	)
+	redisprom.ActiveRequests.With(prometheus.Labels{
+		// We don't have LblClientName and LblDatabase
```

Comment:
I thought h.ClientName is client name, and database can be deduced from the address (if there's no `/n` at the end of address it's `0` or empty, forgot which is the one defined in the spec)

Reply:
I don't understand the second part of your statement.

As for `h.ClientName`, it refers to the pool and isn't being passed down to Redis to identity the client. So it is the pool.

Reply:
from my understanding (and I got that from the spec pr discussion with kyle and nate), redis' "database" is basically something like this:

"ip:port/n"

while by default n=0 and you can omit it as "ip:port" (which is the vast majority of our usages)

I _believe_ (but I could totally be wrong) that n must be an int (vs. free form string)

Reply:
is this the same thing as `redis.Options.DB` ?

Reply:
looks so but I'm not 100% sure. you probably need someone more familiar with redis to confirm it 😅

Reply:
> MAY be blank if not specified.

Can I just leave it blank ?